eo-parser/src/test/java/org/eolang/parser/RedundantParenthesesTest.java

```diff
@@ -96,7 +96,8 @@ static Stream<Arguments> testCases() {
             Arguments.of("[] > foo\n  ((1.add 1)) > x", false),
             Arguments.of("[] > foo\n  1.add 1 > x\n  (1.add 1)", false),
             Arguments.of("[] > foo\n  1.add 1 > x\n  (1.add 1) > y", false),
-            Arguments.of("[] > foo\n  add > x\n    (1)\n    (1)\n", false)
+            Arguments.of("[] > foo\n  add > x\n    (1)\n    (1)\n", false),
+            Arguments.of("# Comment().\n[] > obj\n", true)
```

Comment:
@maxonfjvipon As I understand, the `correct` examples go first before the `incorrect` examples:
```
true
true
...
false,
false
```
You have:
```
true
true
...
false,
false,
true
```
Maybe it's better to move the last statement higher?

Reply:
@volodya-lombrozo sure